### DISCUSSION TITLE
Add full dates for deceased

### DISCUSSION THREAD

**Discussion by mauvehed** [restincode/restincode#203] [Ideas]
In prep for the twitter bot (#9), we need the full dates of as many people as possible so we can schedule tweets on anniversaries.

**Comment by attritionorg** [comment]
Ideally, birth date as well. Those are often mentioned in obits, so finding those for each person would be great.

**Comment by mauvehed** [comment]
Following up on this, cause we still need it, is a standardised format for the date for easy parsing.

I’m thinking seconds since EPOCH.
